modules/deployment-scripts/create-kv-certificate/create-kv-cert.sh

```diff
@@ -14,7 +14,7 @@
         policy=$(az keyvault certificate get-default-policy \
             | sed -e s/CN=CLIGetDefaultPolicy/CN=${certCommonName}/g )
     else
-      if [ "$issuerProvider" == "DigiCert" ] || [ "$issuerProvider" == "GlobalCert"]; then
+      if [ "$issuerProvider" == "DigiCert" ] || [ "$issuerProvider" == "GlobalCert" ]; then
         az keyvault certificate issuer create \
           --vault-name $akvName \
```

Comment:
📝 **[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Double quote to prevent globbing and word splitting. [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)